dashboard/db/schema.rb

```diff
@@ -1702,7 +1703,7 @@
     t.index ["storage_app_id"], name: "index_project_commits_on_storage_app_id"
   end
 
-  create_table "projects", id: :integer, charset: "utf8mb4", collation: "utf8mb4_general_ci", force: :cascade do |t|
+  create_table "projects", id: :integer, charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
```

Comment:
Not sure we want this change?

Reply:
oh weird - thanks! Will revert this line.

Reply:
This line has updated unexpectedly in the past locally after I run a new migration. Does it ever happen to you?

Reply:
mmm yes that sounds semi-familiar...quick google suggests what you've got might be the default for mysql8 now?

https://dev.mysql.com/blog-archive/mysql-8-0-collations-migrating-from-older-collations/